dagmc/dagnav.py

```diff
@@ -238,13 +272,16 @@ def _get_triangle_sets(self):
 
 class Group(DAGSet):
 
+    def __contains__(self, ent_set: DAGSet):
+        return any(vol.handle == ent_set.handle for vol in self.get_volumes().values())
```

Comment:
Love this. Really slick. :)

It should probably account for groups that hold surfaces too though, which can be stored in groups representing boundary conditions.